src/ReadCSV.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Created by kenshoo-linux on 14/07/16.
+ */
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadCSV {
+
+    public static void main(String[] args) {
+
+        ReadCSV obj = new ReadCSV();
+        obj.run();
+
+    }
+
+    public void run() {
+
+        String csvFile = "/home/kenshoo-linux/develop/projects/corejavaexercise (copy)/macycExampleFile.csv - macycExampleFile.csv.csv";
+        BufferedReader br = null;
+        String line = "";
```

Comment:
Try to declare variable in the closet point you use them for. Also no need to init to empty string.